sql/ddl_api.sql

```diff
@@ -17,23 +17,26 @@ CREATE OR REPLACE FUNCTION  create_hypertable(
     number_partitions       INTEGER = NULL,
     associated_schema_name  NAME = NULL,
     associated_table_prefix NAME = NULL,
-    chunk_time_interval     BIGINT =  _timescaledb_internal.interval_to_usec('1 month'),
+    chunk_time_interval     BIGINT = NULL,
```

Comment:
Is it possible to add a version of this function that takes an interval instead of a BIGINT? That would allow people that use TIMESTAMP dimensions to simply specify their `chunk_time_interval` as `interval '1 week'`. This version of the function would throw an error on INTEGER type columns and point to BIGINT-version.